Tests.Vpn.Service/testdata/Build-Assets.ps1

```diff
@@ -0,0 +1,68 @@
+$errorActionPreference = "Stop"
+
+Set-Location $PSScriptRoot
+
+# If hello.go does not exist, write it. We don't check it into the repo to avoid
+# GitHub showing that the repo contains Go code.
+if (-not (Test-Path "hello.go")) {
```

Comment:
Is there a way we can avoid taking a dependency on Go here? E.g. write the hello program in C#?

Reply:
Since we don't have to recompile it ever again (unless we want to add new test cases) as the binaries are checked in, I'd like to just leave it as Go. Originally I wrote an assembly program so I could keep it as slim as possible but it couldn't get signed by JSign for some reason so I went with Go instead to keep it simple and small.

If we went with C# we'd need to add a new project to the solution which seems overkill

Reply:
I guess this makes it the most similar to the binary we will actually be checking.